<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemPropertiesWidget</class>
 <widget class="QWidget">
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QGridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="itemDAMLabel"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="itemDAMText"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="itemPrePowerLabel">
       <property name="text">
        <string>Prefix:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="itemPrePowerText"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="itemSufPowerLabel">
       <property name="text">
        <string>Suffix:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="itemSufPowerText"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="itemUniquePower1Label"/>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="itemUniquePower1Text"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="itemUniquePower2Label"/>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="itemUniquePower2Text"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="itemUniquePower3Label"/>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="itemUniquePower3Text"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="itemUniquePower4Label"/>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="itemUniquePower4Text"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="itemUniquePower5Label"/>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="itemUniquePower5Text"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="itemUniquePower6Label"/>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="itemUniquePower6Text"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="itemRequirementsLabel">
       <property name="text">
        <string>Req.:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="itemRequirementsText"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
